geomesa-utils-parent/geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/iterators/PlaybackIterator.scala

```diff
@@ -71,47 +57,25 @@ class PlaybackIterator(
     case t => t.indexOf(dtgName)
   }
   require(dtgIndex != -1, "Invalid date field")
-  private val dtgProp = ff.property(dtgName)
-  private val sort = ff.sort(dtgName, SortOrder.ASCENDING)
-
-  private val windowMillis = window.map(_.toMillis).getOrElse(interval._2.getTime - interval._1.getTime + 1)
-
   private var start: Long = -1
   private var eventStart: Long = -1
 
-  private val features = new LinkedBlockingQueue[SimpleFeature](readAhead)
-  private var staged: SimpleFeature = _
-
-  private val executor = Executors.newSingleThreadExecutor()
-  executor.submit(new QueryRunnable())
-
-  override def hasNext: Boolean = {
-    if (staged != null) {
-      true
-    } else {
-      staged = features.take()
-      if (!PlaybackIterator.terminal.eq(staged)) {
-        true
-      } else {
-        features.put(staged) // re-queue the terminal value to keep this method idempotent
-        staged = null
-        false
-      }
-    }
-  }
+  override def hasNext: Boolean = iterator.hasNext
 
   override def next(): SimpleFeature = {
-    val feature = staged
-    staged = null
+    val feature = iterator.next();
+    logger.warn("Retrieving {}", feature.getAttribute("name"))
```

Comment:
```suggestion
```
should probably just delete this line, unless you want to switch it to `feature.getID` (since not everything will have a name)